static/js/settings_account.js

```diff
@@ -304,6 +304,21 @@ exports.set_up = function () {
         }
     });
 
+    $("#user_export_data").on('click', function (e) {
+        e.preventDefault();
+        e.stopPropagation();
+        channel.post({
+            url: "/json/users/me/export",
+            success: function (response) {
+                ui_report.success(i18n.t(response.msg), $('#user_export_data_message'));
+            },
+            error: function (xhr) {
+                ui_report.error(i18n.t("Could not start export"), xhr, $('#user_export_data_message'));
+            },
+        });
+    });
```

Comment:
We might want to replace the `ui_report` stuff with more of a loading indicator/success/fail thing like we've done with other settings.

Reply:
Do you mean we should show the status of the export? Or just the status of the request to start the export? 

Also, it looks like the [indicator message is hard-coded](https://github.com/zulip/zulip/blob/master/static/js/settings_ui.js#L32). Would you like me to refactor that?

Reply:
The status of the export, I think?  That's what a user would care about.

Yeah, refactoring that in a commit on the bottom would be a good idea.